### ISSUE TITLE
CSS for config/`qute://` pages

### ISSUE THREAD

**Issue by adigitoleo** [qutebrowser/qutebrowser#7116]
It would be nice to have CSS for the internal pages (`qute://<stuff>`), which should support `prefers-color-scheme: dark`. It is possible to set `colors.webpage.bg` to something dark, but this makes the text unreadable.

I also found that opening a settings page from the `:help` prompt does not respect `colors.webpage.bg` anyway.

**Comment by The-Compiler** [comment]
`qute://config` recently got a new CSS by @twigleingrid - but that indeed doesn't support dark styling. I suppose it should! As for the others, there are some design improvements under way for those already, but they are probably used much less anyways.

**Comment by adigitoleo** [comment]
Yes for me the main thing is to be able to read the help page when I'm using (CSS) dark mode. I will see if I get time to look at this on the weekend.

**Comment by adigitoleo** [comment]
> qute://config

I guess you mean `qute://settings`? I didn't know it existed tbh :smile:, mostly I just use the prompt or config file. Now this makes me wonder if `qute.css` is being applied anywhere else? Help pages don't seem to set a background color...

**Comment by The-Compiler** [comment]
Err, yes, I did indeed mean `qute://settings`. `www/qute.css` is only relevant for the qutebrowser.org website. Most `qute://` pages use the CSS embedded in `qutebrowser/html/styled.html`. The help pages are different, as those are currently just generated by asciidoc, with the style coming from there. I suppose this will change once I got a chance to look at #345 finally - then, I'd likely want to use [Furo](https://pradyunsg.me/furo/) for those.

**Comment by adigitoleo** [comment]
See https://github.com/qutebrowser/qutebrowser/pull/7221#issuecomment-2201681505 for a workaround on QtWebEngine 6.7+
